py-polars/tests/test_df.py

```diff
@@ -350,6 +350,33 @@ def test_init_records() -> None:
     assert df_xy.rows() == [(1, 2), (2, 1), (1, 2)]
 
 
+def test_struct_cols() -> None:
+    """Test that struct columns can be imported and work as expected."""
+
+    def build_struct_df(data):
```

Comment:
The build is failing on type checking. Do you need to add typing here? Try running `make pre-commit` in the `py-polars/` directory.